tests/Moq.Analyzers.Test/InSequenceSetupShouldBeProperlyConfiguredAnalyzerTests.cs

```diff
@@ -0,0 +1,82 @@
+using Moq.Analyzers.Test.Helpers;
+using Verifier = Moq.Analyzers.Test.Helpers.AnalyzerVerifier<Moq.Analyzers.InSequenceSetupShouldBeProperlyConfiguredAnalyzer>;
+
+namespace Moq.Analyzers.Test;
+
+public class InSequenceSetupShouldBeProperlyConfiguredAnalyzerTests
+{
+    public static IEnumerable<object[]> TestData()
+    {
+        // Test cases that should NOT produce diagnostics (valid InSequence usage)
+        IEnumerable<object[]> validUsage = new object[][]
+        {
+            // Valid InSequence with MockSequence followed by Setup
+            ["""
+             var sequence = new MockSequence();
+             var mock = new Mock<IService>();
+             mock.InSequence(sequence).Setup(x => x.DoSomething());
+             """],
+
+            // Multiple InSequence calls with same sequence
+            ["""
+             var sequence = new MockSequence();
+             var mock1 = new Mock<IService>();
+             var mock2 = new Mock<IService>();
+             mock1.InSequence(sequence).Setup(x => x.DoSomething());
+             mock2.InSequence(sequence).Setup(x => x.DoOtherThing());
+             """],
+
+            // InSequence with variable
+            ["""
+             var sequence = new MockSequence();
+             var mock = new Mock<IService>();
+             var inSequenceMock = mock.InSequence(sequence);
+             inSequenceMock.Setup(x => x.DoSomething());
+             """],
+        }.WithNamespaces().WithMoqReferenceAssemblyGroups();
+
+        // Test cases that SHOULD produce diagnostics (invalid InSequence usage)
+        IEnumerable<object[]> invalidUsage = new object[][]
+        {
+            // InSequence without proper MockSequence parameter (null) -- this is a compile error, not an analyzer warning
```

Comment:
If this is a compiler error, does the test actually run? The test harness needs valid code to run analyzers on